scripts/create_perf_json.py

```diff
@@ -327,6 +327,13 @@ def get(key: str) -> str:
         if self.umask:
             self.umask = self.umask.split(",")[0]
             umask_ext = get('UMaskExt')
+
```

Comment:
This change will apply the fix to more than just IIO. I had the following:
```
            if umask_ext and self.unit != 'IIO':
                self.umask = umask_ext + self.umask[2:]
```

Reply:
Thanks Ian, that certainly works as well. I'm getting the same output with both approaches, and yours is certainly simpler.

`PortMask` and `FCMask` might be removed on subsequent platforms. We could revisit this logic at that point.

Reply:
Thanks for checking! I'm happy to drop my fix in favor of this.

Reply:
Thanks Ian. Hopefully these changes don't introduce too many conflicts in your local `create_perf_json.py` version!